configs/firefox/README.md

```diff
@@ -0,0 +1,55 @@
+# firefox configuration readme 
+  * enable_create_new_profile : `true/false, will create new firefox profile before running the whole suite` 
```

Comment:
* enable_create_new_profile : `true/false, it will create new Firefox profile before running the whole suite if enabled`